languages/go/INSTRUCTIONS.md

```diff
@@ -0,0 +1,46 @@
+
+The Bitwarden SDK is written in Rust, and we utilize FFI via [cgo](https://pkg.go.dev/cmd/cgo). We currently support static linking the Bitwarden Go module on Linux (x86-64 & arm64), Mac (x86-64 & arm64), and Windows (x86-64).
+
+Please see the main [README](./README.md) and [example](./example/example.go) for examples.
+
+## Linux
+
+1. Make sure you have the following on your system:
+    - [Go](https://go.dev/dl)
+    - A C/C++ toolchain. We recommend the [MUSL toolchain](https://musl.libc.org). You can install this on most debian based systems with: `sudo apt install musl-tools`
+2. Verify cgo is enabled and `CC` is set with: `go env`.
+    - You can enable cgo with: `go env -w CGO_ENABLED=1`
+    - You can set `CC` with: `go env -w CC=musl-gcc`
+3. Make sure you have added the Bitwarden Go SDK with go get: `go get github.com/bitwarden/sdk-go`
+4. You can build your project with the appropriate libraries linked statically with: `go build -ldflags '-linkmode external -extldflags "-static -Wl,-unresolved-symbols=ignore-all"' -o myapp`, where `myapp` is the name of the output binary.
+
+## Mac
+
+1. Make sure you have the following on your system:
+    - [Go](https://go.dev/dl)
+    - A C/C++ toolchain. [Clang](https://clang.llvm.org/get_started.html) is the default Xcode compiler on Mac OS. The easiest way to ensure you have the toolchain is to install the Xcode Command Line tools.
+        - You can install them with: `xcode-select -–install`
+2. Verify cgo is enabled and `CC` and `CXX` are set with: `go env`.
+    - You can enable cgo with: `go env -w CGO_ENABLED=1`
+    - You can set `CC` and `CXX` with: `go env -w CC=clang CXX=clang++`
+3. Make sure you have added the Bitwarden Go SDK with go get: `go get github.com/bitwarden/sdk-go`
+4. Your project should build with: `go build`
+
+## Windows
+
+1. Make sure you have the following on your system:
+    - [Go](https://go.dev/dl)
+    - GCC via [MSYS2](https://www.msys2.org)
+        - GCC is required on Windows according to [this cgo](https://go.dev/wiki/cgo) page. Microsoft recommends using the MinGW-w64 toolchain via [MSYS2](https://www.msys2.org) for up-to-date native builds of GCC [here](https://code.visualstudio.com/docs/cpp/config-mingw#_installing-the-mingww64-toolchain). MSYS2 installation instructions can be found [here](https://www.msys2.org).
+        - After following the MSYS2 installation instructions, you should be able to run the following to verify everything is installed:
+            - `gcc --version`
+            - `g++ --version`
+            - `gdb --version`
+2. Verify cgo is enabled and `CC` and `CXX` are set with: `go env`.
+    - You can enable cgo by setting the flag: `go env -w CGO_ENABLED=1`
+    - You can set `CC` and `CXX` with: `go env -w CC=gcc CXX=g++`
+3. Make sure you have added the Bitwarden Go SDK with go get: `go get github.com/bitwarden/sdk-go`
+4. Your project should build with: `go build`
```

Comment:
What about breaking the steps for each OS down into a few sections? 

I personally find this easier to read and code blocks are more convenient to copy.

Windows example below:






## Windows

### Prerequisites

- [Go](https://go.dev/dl)
- GCC

Go [documentation](https://go.dev/wiki/cgo) recommends installing the MinGW-w64 gcc compiler.

We recommend following the Visual Studio Code [guide](https://code.visualstudio.com/docs/cpp/config-mingw#_installing-the-mingww64-toolchain) to install the MinGW-w64 toolchain.

### Set Go Environment for cgo

#### Enable cgo

```shell
go env -w CGO_ENABLED=1
```

#### Set C and C++ Compilers

```shell
go env -w CC=gcc CXX=g++
```


Verify settings by running:

```shell
go env
```

### Install the Bitwarden Go SDK

Add the Bitwarden Go SDK to a project with the following command:

```shell
go get github.com/bitwarden/sdk-go
```

### Build

Validate the project builds successfully

```shell
go build
```

Reply:
Good point, definitely more readable and more concise.

Updated in: https://github.com/bitwarden/sdk/pull/861/commits/df7c5cd8195b8b0f0adc717ecc978277ead5cb78.